git/foundations_git/git_basics.md

```diff
@@ -180,7 +180,7 @@ With that done, you can now choose to use either `git commit -m "your message he
 
 To make a commit with Visual Studio Code as the text editor, just type `git commit`. After you hit <kbd>Enter</kbd> a new tab in VS Code will open for you to write your commit message. You may provide more details on multiple lines as part of your commit message. After typing your commit message, save it <kbd>Ctrl</kbd> + <kbd>S</kbd> (Mac: <kbd>Cmd</kbd> + <kbd>S</kbd>) and close the tab. If you return to the command line, you will see your commit message and a summary of your changes.
 
-### Conclusion
+#### Conclusion
```

Comment:
I honestly feel like we should remove this section, as it isn't really a sub-section of the assignment section and we don't typically offer "conclusion" sections in lessons.